admin_manual/configuration_files/big_file_upload_configuration.rst

```diff
@@ -2,21 +2,18 @@
 Uploading big files > 512MB
 ===========================
 
-The default maximum file size for uploads is 512MB. You can increase this 
-limit up to what your filesystem and operating system allows. There are certain 
-hard limits that cannot be exceeded:
+The default maximum file size for uploads, in ownCloud, is 512MB. 
+You can increase this limit up to the maximum file size which your filesystem, operating system, or other software allows, for example:
 
-* < 2GB on 32Bit OS-architecture
+* < 2GB on a 32Bit OS-architecture
 * < 2GB on Windows (32Bit and 64Bit)
 * < 2GB with Server Version 4.5 or older
 * < 2GB with IE6 - IE8
 * < 4GB with IE9 - IE11
+ 
+Please consult the documentation for your filesystem.
```

Comment:
From my POV, changes are ok

You still write ...for your filesystem...
I think it is better to write ...of your environment...
See http://www.ntfs.com/ntfs_vs_fat.htm
If your file source system is FAT, the limit is before owncloud, you cant have a bigger file existing
If your target filesystem is mounted via SMB it is a matter of the environment (SMB...)
Same for any network protocol accessing a filesystem
If you have eg ext4, than the limit is in the range of TB

Just my 2c :smile: